src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! Library Template for Rust
+//! This is a crate description, needed or else missing_docs warning will occur
+
+/// Adds one to a number
+///
+/// # Arguments
+///
+/// * `val` - Any U8 number
+///
+pub fn add_one(val: u8) -> u8 {
+    val + 1
```

Comment:
Another styling thing to consider. Although `return` is not necessary in this case, should we enforce it? We don't want to have a codebase that has a mix use of how things returned.

Reply:
It's idiomatic Rust to not use "return" and treat functions like expressions. This is most apparent in `match` examples.

```rust
let binary = match boolean {
        false => 0, // no explicit "return", but returns
        true => 1,
    };
```

I'll leave it as is for now